.dockerignore

```diff
@@ -14,3 +14,4 @@ junit.xml
 npm-debug.log
 static-analysis
 setupEnzyme.js
+node_modules
```

Comment:
Docker needs to ignore `node_modules` because we *must* run `npm install` and `npm run build` within the Docker build. This is true because `process.env` values are fixed at **build time** (by `webpack`), so if we run `npm install` and `npm run build` locally and deploy what's in `node_modules` and `dist/`, the environment variables will be set to their values from our individual development environments.